### PR TITLE
Cosmetic changes in lpc_asm.nasm and utf8.c

### PR DESCRIPTION
**Pull request by lvqcl**
src/libFLAC/ia32/lpc_asm.nasm:
a) fix comment about possible values of `lag` variable
b) fix indentation of comments

src/share/utf8/utf8.c:
a) remove extra lines to make the code look better
b) remove trailing whitespaces

### CODE REVIEW

File: src/libFLAC/ia32/lpc_asm.nasm [hunk 1]
```diff
@@ -637,7 +637,7 @@ cident FLAC__lpc_compute_autocorrelation_asm_ia32_sse_lag_16_old
 	sub	esp, 32
 
 	;ASSERT(lag > 0)
-	;ASSERT(lag <= 12)
+	;ASSERT(lag <= 16)
```
[the remark this answers is not in the document]
Comment (lvqcl) [reply src/libFLAC/ia32/lpc_asm.nasm]: Yes, it's just a comment in the form of a C code.

There are 4 asm functions and 8 function with intrinsics that compute autocorrelation, they are  named as

`FLAC__lpc_compute_autocorrelation_......._sse_lag_NN_......`

and each has comment or assertion `FLAC__ASSERT(lag <= NN)`, except this one.

It seems that I copy-pasted some text from `...asm_ia32_sse_lag_12` function (see deb209906c28e6d35ef2c47a94285d7acc967c1b) and forgot to change the value from 12 to 16.

**Comment by erikd** [comment]
Thanks!

**Comment by erikd** [comment]
Once it has been merged it can't be reviewed. However, by merging it, i showed my approval :)

**Comment by lvqcl** [comment]
I wanted to say that for some reason I can't see it in the list of commits. I also just downloaded the latest snapshot, and this patch is also not there...

**Comment by erikd** [comment]
It seems to be there now right? https://github.com/xiph/flac/commits/master

**Comment by lvqcl** [comment]
Yes, here and also [there](https://git.xiph.org/?p=flac.git;a=summary) -  everything is ok.

**Pull request merged**